act-rules-format.bs

```diff
@@ -89,17 +89,40 @@ An ACT Rule MUST have a description that is in plain language and provides a bri
+
+[=Atomic rules=] and [=composite rules=] SHOULD identify the accessibility requirements that are *not satisfied* when the outcome of a rule is Fail. An ACT Rule is a complete or partial test for one or more accessibility requirements. This means that most ACT rules test only part of an accessibility requirement, but it MUST NOT test more than the accessibility requirement it lists.
+
+Because ACT Rules often have a smaller scope than the accessibility requirement they test, passing a rule does not necessarily mean that the accessibility requirement is *satisfied*. ACT Rules MUST indicate whether or not they can be used to determine that the accessibility requirement is *satisfied*.
+
+<div class=example>
+  <p>The following rule could **not** be used to satisfy [WCAG 2.1 Success criterion 2.4.2 Page Titled](https://www.w3.org/TR/WCAG21/#page-titled)</p>
+  <blockquote>
+    <p>This rule tests that an HTML page has a `title` element.</p>
+  </blockquote>
 </div>
 
-Accessibility requirements are just that: A requirement that a particular web page must conform to for it to be considered accessible. This can (and usually does) include the WCAG success criteria. Often organizations have additional requirements which may come from different sources, such as local laws, internal standards, etc. These too are considered accessibility requirements and can be tested using the ACT Rules Format. What the precise requirements are for any test is beyond the scope of the ACT Rules Format.
+<div class=example>
+  <p>The following rule could be used to satisfy [WCAG 2.1 Success criterion 2.4.2 Page Titled](https://www.w3.org/TR/WCAG21/#page-titled)</p>
+  <blockquote>
+    <p>This rule tests that the first `title` element of an HTML page describes the topic or purpose of that page.</p>
+  </blockquote>
+</div>
+
+Outcomes from an ACT Rule MUST be consistent with the accessibility requirement, e.g. a rule only returns the outcome Fail when the content does *not satisfy* the accessibility requirement. If an ACT Rule indicates it can be used to satisfy the accessibility requirement, a Pass or Inapplicable result for that rule MUST satisfy the accessibility requirement.
+
+This means that the rule maps to the accessibility requirement, as opposed to it merely being related to the requirement, thematically or otherwise. Because of this, atomic rules used in composite rules often do not map to any accessibility requirement. Failing the composite rule fails the accessibility requirement, but failing any of its atomic rules may not. In such cases the atomic rules MUST NOT list the accessibility requirements. These could be provided as background information instead.
 
```

Comment:
IMO this part is quite confusing: "MUST be consistent" only applies to Fail or, if the rule indicates it applies to the entire requirement, also for Pass. Inapplicable is an exception anyway because WCAG doesn't define that. But if it is an atomic rule then also the Fail can be inconsistent, and you have to look at it in context of the composite rule.

Not sure what to do about this but it seems we are trying to mix *two* outputs rather than one: (1) the outcome of the rule, and (2) any implications this outcome could have on meeting the accessibility requirement. Maybe we need to have two such outputs? The outcome of the rule is *always* returned, and the other is conditional and must be consistent with the accessibility requirement.